homeassistant/components/apsystems/strings.json

```diff
@@ -3,19 +3,28 @@
     "step": {
       "user": {
         "data": {
-          "host": "[%key:common::config_flow::data::host%]",
-          "username": "[%key:common::config_flow::data::username%]",
-          "password": "[%key:common::config_flow::data::password%]"
+          "ip_address": "[%key:common::config_flow::data::ip%]"
         }
       }
     },
     "error": {
-      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
-      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
-      "unknown": "[%key:common::config_flow::error::unknown%]"
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
     },
     "abort": {
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "total_power": { "name": "Total Power" },
+      "total_power_p1": { "name": "Power of P1" },
+      "total_power_p2": { "name": "Power of P2" },
+      "lifetime_production": { "name": "Total Lifetime Production" },
+      "lifetime_production_p1": { "name": "Lifetime Production of P1" },
+      "lifetime_production_p2": { "name": "Lifetime Production of P2" },
+      "today_production": { "name": "Production of Today" },
+      "today_production_p1": { "name": "P1's Production of Today" },
+      "today_production_p2": { "name": "P2's Production of Today" }
+    }
```

Comment:
Sentence case is not fixed yet

Reply:
Sorry, wasn't aware of the sentence case meaning (even though it is pretty self-explanatory).